server/routes/roomsRoutes.js

```diff
@@ -12,5 +13,4 @@ router.delete('/:id', roomsController.deleteRoom);
 
```

Comment:
In order to make this file cleaner I would recommend to use the router function "route", as it is made in routes/messages.js. We should use similar names for our files too

Reply:
Changed and pushed to feature/rooms-setup 👌